### PR TITLE
feat: Additional info v3

### PR DESCRIPTION
**Pull request by harshil1793**
## Chromatic

https://1695-additional-info-v3--60f9b557105290003b387cd5.chromatic.com

## Description
This pull request adds a new additional info component. CSS sizes defined in rem units to px units
Sketch: https://www.sketch.com/s/610156b6-f281-4497-81f3-64454fc72156/symbols?g=Additional%2520info%252Fuswds-DO%2520NOT%2520USE%252FMVP#Version
Closes [1695](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1695)

## Testing done
Storybook
Chrome/Edge/Mozilla

## Screenshots
additional info

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by powellkerry — APPROVED** [review]

File: packages/web-components/src/components/va-additional-info/va-additional-info.tsx [hunk 1]
```diff
@@ -96,25 +100,52 @@ export class VaAdditionalInfo {
...
+            tabIndex={0}
+            onClick={this.toggleOpen.bind(this)}
+            onKeyDown={this.handleKeydown.bind(this)}
+          >
+            <div>
+              <span class="additional-info-title">{this.trigger}</span>
+              <i class="fa-angle-down" role="presentation" />
+            </div>
+          </a>
+          <div id="info" class={`${infoClass}`}>
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-additional-info/va-additional-info.tsx:123]: Can this just be `class={infoClass}`?

File: packages/web-components/src/components/va-additional-info/va-additional-info.tsx [hunk 2]
```diff
@@ -96,25 +100,52 @@ export class VaAdditionalInfo {
...
+      const infoClass = classnames({
+        'open': this.open,
+        'closed': !this.open
+      });
+      return (
+        <Host>
+          <a
+            role="button"
+            aria-controls="info"
+            aria-expanded={this.open ? 'true' : 'false'}
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-additional-info/va-additional-info.tsx:113]: If we're going to start using the `classnames` package, could it be used here for the aria-expanded too?

**Comment by jamigibbs** [comment]
If we are sharing the CSS between v1 and v3, do we think we should convert all of the existing `rem` to `px` while we're adding this variation? 

For example: https://github.com/department-of-veterans-affairs/component-library/blob/1695-additional-info-v3/packages/web-components/src/components/va-additional-info/va-additional-info.css#L47-L50

**Comment by jamigibbs** [comment]
Do we want to update this to `scss` now (I guess in anticipation of us needing sass files at some point?) https://github.com/department-of-veterans-affairs/component-library/blob/b6a0dbdf044ccb3937ea779441eabbf0da5ab50d/packages/web-components/src/components/va-additional-info/va-additional-info.tsx#L21

Let's also make sure we're adding Lilly or Barb for a design review on this PR.

**Comment by jamigibbs** [comment]
I realize that we are not needing to import any USWDS modules but I wonder if we should add the `@forward 'settings';` to the stylesheet like we have in the other USWDS variation components.

https://github.com/department-of-veterans-affairs/component-library/blob/1695-additional-info-v3/packages/web-components/src/components/va-button/va-button.scss#LL1C1-L1C21

File: packages/web-components/src/components/va-additional-info/va-additional-info.tsx (hunk 2 above)
Comment (harshil1793) [reply packages/web-components/src/components/va-additional-info/va-additional-info.tsx:113]: yea I agree, I just thought to optimize it a little but I can revert back too

**Comment by harshil1793** [comment]
I initially converted the files to `SCSS`, but since we didn't make any alterations thereafter, I refrained from pushing it.
I'm not sure what `@forward 'settings';` is specifically used for here.

**Comment by jamigibbs** [comment]
> I initially converted the files to `SCSS`, but since we didn't make any alterations thereafter, I refrained from pushing it. I'm not sure what `@forward 'settings';` is specifically used for here.

It will forward the [USWDS settings we have here](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/global/_settings.scss) for font stack and uswds image path.

```
@use "uswds-core" with ($theme-show-notifications: false,
  $theme-image-path: '../../../../../node_modules/@uswds/uswds/dist/img/',
  $theme-font-type-sans: 'source-sans-pro', // Formation override.
  $theme-font-sans-custom-stack: '"Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans', // Formation override.
);
```

I am honestly not sure the point of having this component as a "USWDS v3" component if there is nothing different between it and the original v1.

**Comment by jamigibbs** [comment]
> I am honestly not sure the point of having this component as a "USWDS v3" component if there is nothing different between it and the original v1.

From Matt in our Slack channel: 

> there are no anticipated [design] updates in the immediate future. There might potentially be color differences between the versions but perhaps not even that.

**Review by jamigibbs — APPROVED** [review]

**Review by jamigibbs — APPROVED** [review]
Before merging, can you update the web components package version?

**Pull request merged**